api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java

```diff
@@ -542,7 +542,7 @@ public void shouldCopyTranslationsFromImportedForm() throws IllegalAccessExcepti
 		assertEquals(importedTranslationsJson, FileUtils.readFileToString(translationFile));
 	}
 
-	private static FormTranslation createFormTranslation(String locale, String version, String formName) {
+	public static FormTranslation createFormTranslation(String locale, String version, String formName) {
```

Comment:
Why has this been made public?

Reply:
We are using the same method in BahmniFormServiceImplTest.java, so for reusing it, we made it public.

Reply:
Can we move it to a util or helper class within test?